python/dgl/graphbolt/datapipe_utils.py

```diff
@@ -0,0 +1,81 @@
+"""DataPipe utilities"""
+
+# pylint: disable=unused-import
+try:
+    from torchdata.dataloader2.graph import traverse_dps
+
+    DATAPIPE_COMPATIBILITY = "0.5"
+except ImportError:
+    # PyTorch 1.12-
+    from torchdata.dataloader2.graph import traverse
+
+    DATAPIPE_COMPATIBILITY = "0.4"
+
+    def traverse_dps(datapipe):
+        """Wrapper of PyTorch 1.12 ``traverse`` function to PyTorch 1.13
```

Comment:
pytorch 1.13 is enabled in CI. please have a try

Reply:
Removed.